docs/app/Examples/collections/Form/FieldVariations/FormExampleRequiredField.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Form, Input } from 'semantic-ui-react'
+
+const FormExampleRequiredField = () => (
+  <Form>
+    <Form.Field required>
+      <label>Last name</label>
+      <Input placeholder='Full name' />
+    </Form.Field>
+    <Form.Checkbox
+      inline
+      label='I agree to the terms and conditions'
+      required
+    />
```

Comment:
Could we split these into two examples with separate descriptions?
***
A Form.Field can be required.
```jsx
<Form.Field required>
```

Form.Field shorthand can also be required.
```jsx
<Form.Checkbox ... required />
```